core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageFeatureStep.java

```diff
@@ -53,7 +53,7 @@ void addExportsToNativeImage(BuildProducer<JPMSExportBuildItem> features) {
         features.produce(new JPMSExportBuildItem("org.graalvm.sdk", "org.graalvm.nativeimage.impl", null,
                 GraalVM.Version.VERSION_23_1_0));
         features.produce(new JPMSExportBuildItem("org.graalvm.nativeimage", "org.graalvm.nativeimage.impl",
-                GraalVM.Version.VERSION_23_0_0));
+                GraalVM.Version.VERSION_23_0_999));
```

Comment:
Wouldn't it be easier if the version here was the version from where we need to add this, rather than the version before?

I.e. you would use `23.1.0` here.

I will merge this but I wonder if we should change the behavior in the future.

Reply:
Yes, that makes sense. I can prepare a PR for it.

Reply:
See https://github.com/quarkusio/quarkus/pull/36063